### PR TITLE
Refresh network-evertz-fc7800-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-evertz-fc7800-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 1]
```diff
@@ -1,49 +1,398 @@
 ---
 id: network-evertz-fc7800-snmp
-title: Evertz FC7800
+title: Evertz FC7800 SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 2]
```diff
@@ -1,49 +1,398 @@
...
 
-### Centreon Plugin
+Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Evertz FC7800**
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:10]:
```suggestion
Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Evertz FC7800 SNMP** 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 3]
```diff
@@ -1,49 +1,398 @@
...
+depuis l'interface web et le menu **Configuration > Gestionnaire de connecteurs de supervision** :
+* [Base Pack](./base-generic.md)
 
-Install this plugin on each needed poller:
+## Contenu du pack
 
-``` shell
+### Modèles
+
+Le connecteur de supervision **Evertz FC7800** apporte un modèle d'hôte :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:18]:
```suggestion
Le connecteur de supervision **Evertz FC7800 SNMP** apporte un modèle d'hôte :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 4]
```diff
@@ -1,49 +1,398 @@
...
+| count_psu | count |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Afin de superviser votre **Evertz FC7800** en SNMP, il est nécessaire de configurer l'agent sur l'équipement.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:90]: prérequis standard

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 5]
```diff
@@ -1,49 +1,397 @@
 ---
 id: network-evertz-fc7800-snmp
-title: Evertz FC7800
+title: Evertz FC7800 SNMP
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Connector Dependencies
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:8]:
```suggestion
## Connector dependencies
```

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 6]
```diff
@@ -1,49 +1,397 @@
...
+
+The following monitoring connectors will be installed when you install the **Evertz FC7800** connector through the
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:10]:
```suggestion
The following monitoring connectors will be installed when you install the **Evertz FC7800 SNMP** connector through the
```

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 7]
```diff
@@ -1,49 +1,397 @@
...
+**Configuration > Monitoring Connector Manager** menu:
+* [Base Pack](./base-generic.md)
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **Evertz FC7800** brings a host template:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:18]:
```suggestion
The Monitoring Connector **Evertz FC7800 SNMP** brings a host template:
```

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 8]
```diff
@@ -1,49 +1,397 @@
...
+| count_psu | count |
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:88]: standard prerequisites

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 9]
```diff
@@ -1,49 +1,397 @@
...
+To use this pack, the SNMP service must be properly configured on your **Evertz FC7800** equipment.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:89]:
```suggestion
To use this pack, the SNMP service must be properly configured on your **Evertz FC7800** resource.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:90]:
```suggestion
L'agent SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 10]
```diff
@@ -1,49 +1,398 @@
...
+
+### Flux réseau
+
+La communication doit être possible sur le port UDP 161 depuis le collecteur
+Centreon vers le serveur supervisé.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:95]:
```suggestion
La communication doit être possible sur le port UDP 161 depuis le collecteur Centreon vers le serveur supervisé.
```

File: pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md [hunk 11]
```diff
@@ -1,49 +1,397 @@
...
+
+### Network flow
+
+The target server must be reachable from the Centreon poller on the UDP/161
+SNMP port.
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-evertz-fc7800-snmp.md:94]:
```suggestion
The target server must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

**Pull request merged**